wormhole-connect/src/views/v2/Bridge/SwapInputs/index.tsx

```diff
@@ -74,10 +78,29 @@ function SwapInputs() {
   return (
     <IconButton
       className={classes.swapButton}
+      sx={{
+        animation: `${rotateAnimation} 0.3s linear 1`,
+        '@keyframes spinRight': {
+          '0%': {
+            transform: 'rotate(-180deg)',
+          },
+          '100%': {
+            transform: 'rotate(0deg)',
+          },
+        },
+        '@keyframes spinLeft': {
+          '0%': {
+            transform: 'rotate(180deg)',
+          },
+          '100%': {
+            transform: 'rotate(0deg)',
+          },
+        },
+      }}
       onClick={swap}
       disabled={!canSwap}
     >
-      <SwapVertIcon color="secondary" />
```

Comment:
What's the reason for removing this color?

Reply:
So that the icon can pick up the theme color. Setting it directly was overriding the theme colors that we use in other places.